pydo/database/database.py

```diff
@@ -0,0 +1,42 @@
+from abc import abstractmethod, ABC
+from ..config.config import default_cf_path
+
+
+class Database(ABC):
+    def __init__(self, name, path, config_path=None):
+        self.name = name
+        self.path = path
+        self.connection = None
+        self.complete_path = None
+        if config_path is None:
+            self.config_path = default_cf_path()
```

Comment:
This is a prime place for a ternary 
```suggestion
            self.config_path = default_cf_path() if config_path is None else config_path
```